Mathlib/Combinatorics/SimpleGraph/Coloring.lean

```diff
@@ -454,6 +454,14 @@ theorem cliqueFree_of_chromaticNumber_lt {n : ℕ} (hc : G.chromaticNumber < n)
   rw [← hne] at hc
   simpa using hc
 
+lemma Coloring.surjOn_of_card_le_isClique {γ : Type*} [Fintype γ] {s : Finset V}
```

Comment:
I think the rest of this file uses alpha for the colouring type? If you use that, you can avoid the new local variables